specialTools.lua

```diff
@@ -624,157 +543,21 @@ end
-	-- object.cp.backMarkerOffsetCorrection:		(Distance in meters)	If the implement stops to early or to late, you can specify then it needs to raise and/or turn off the work tool
-	--																	Positive value, moves it forward, Negative value moves it backwards.
-	-- object.cp.frontMarkerOffsetCorrection:	(Distance in meters)	If the implement starts to early or to late, you can specify then it needs to lower and/or turn on the work tool
-	--																	Positive value, moves it forward, Negative value moves it backwards.
-	-- object.cp.haveInversedRidgeMarkerState:	(Boolean)				If the ridmarker is using the wrong side in auto mode, set this value to true
-	-- object.cp.realUnfoldDirectionIsReversed:	(Boolean)				If the tool unfolds when driving roads and folds when working fields, then set this one to true to reverse the folding order.
 	-- object.cp.specialUnloadDistance:			(Distance in meters)	Able to set the distance to the waiting point when it needs to unload. Used by bale loaders. Distance from trailer's turning point to the rear unloading point.
-	-- self.cp.changeDirAngle					(Angle in Degrees)		Overwrite the default automatic direction change angle, used in turn maneuvers.
-	-- self.cp.noStopOnEdge:                    (Boolean)               Set this to true if it dont need to stop the work tool while turning.
-	--																	Some work tool types automatically set this to true.
-	-- self.cp.noStopOnTurn:					(Boolean)				Set this to true if the work tool don't need to stop for 1½ sec before turning.
-	--																	Some work tool types automatically set this to true.
-	-- self.cp.backMarkerOffset:				(Distance in meters)	If the implement stops to early or to late, you can specify then it needs to raise/lower or turn on/off the work tool
-	-- object.cp.noWorkArea						(Boolean)				Some implement have workarea that don't work. This will have set markers skip over thoses tools
-	-- TODO: Add description for all the special varialbes that is usable here.
 
 	courseplay:debug(('%s: askForSpecialSettings(..., %q)'):format(nameNum(self), nameNum(object)), 6);
 
-	local automaticToolOffsetX;
-	-- VEHICLES
-	if self.cp.isGrimmeTectron415 then
-		self.cp.changeDirAngle = 5;
-		self.cp.noStopOnTurn = true;
-		self.cp.noStopOnEdge = true;
-		self.isStrawEnabled = false;
-
-	elseif self.cp.isHolmerTerraDosT4_40 then
-		self.cp.changeDirAngle = 20;
-		self.cp.noStopOnTurn = true;
-		self.cp.noStopOnEdge = true;
-		self.cp.backMarkerOffset = 4.5;
-		self.isStrawEnabled = false;
-	elseif self.cp.isCaseIHA8800MR then
-		self.cp.changeDirAngle = 1;
-	end;
-
-	-- OBJECTS
-	if object.cp.isSP400F then
-		object.cp.backMarkerOffsetCorrection = 0.5;
-		object.cp.frontMarkerOffsetCorrection = -0.25;
-
-	elseif object.cp.isUrsusT127 then
+	if object.cp.isUrsusT127 then
 		object.cp.specialUnloadDistance = -1.8;
```

Comment:
@schwiti6190 do you know if this specialUnloadDIstance is still needed?

Reply:
@pvaiko seems like it might still be needed for reverse or BaleLoaderAIDriver, but I am not sure.